src/Web/View/Base/AbstractView.php

```
@@ -78,7 +80,9 @@
     /**
      * The filesystem instance.
      */
-    public private(set) readonly LocalFilesystem $filesystem;
+    public LocalFilesystem $filesystem {
```

PHPStan (2.0.3)

Syntax error, unexpected T_PUBLIC on line 83


(phpstan.parse)